infra/consistency/src/validators/ident.js

```diff
@@ -206,6 +291,21 @@ async function validateIdentities({
 }) {
   const identityEvents = contractsContext.identityEvents
   const identities = {}
+  const latestEvents = {}
+
+  const web3 = new Web3(
+    'https://eth-mainnet.alchemyapi.io/jsonrpc/FCA-3myPH5VFN8naOWyxDU6VkxelafK6'
```

Comment:
Should we make this configurable?

Reply:
yeah, good catch.  Intended to do that and forgot all about it.  Thanks